internal/metadata/clustermdloader_test.go

```diff
@@ -49,6 +56,60 @@ func TestUpdatePlan(t *testing.T) {
 		}
 	})
 
+	t.Run("check if all clusters in constant were loaded", func(t *testing.T) {
+
+		type Teststruct struct {
+			//Kind string `yaml:"kind"`
+			Metadata struct {
+				Name string `yaml:"name"`
+			} `yaml:"metadata"`
+		}
+
+		p := plantypes.NewPlan()
+		loader := metadata.ClusterMDLoader{}
+		cmMap := loader.GetClusters(p)
+		//fmt.Println(cmMap)
+		files, err := ioutil.ReadDir("clusters/")
```

Comment:
We can reuse the below function to find all yaml files

https://github.com/konveyor/move2kube/blob/8afca8785e03fc85a6859b2ce6652916043cd016/internal/common/utils.go#L46